docs/nlu/components.rst

```diff
@@ -256,23 +256,37 @@ CountVectorsFeaturizer
 Intent Classifiers
 ------------------
 
+
+.. _keyword_intent_classifier:
+
 KeywordIntentClassifier
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-:Short: Simple keyword matching intent classifier. Not intended to be used.
+:Short: Simple keyword matching intent classifier.
 :Outputs: ``intent``
 :Requires: nothing
 :Output-Example:
 
     .. code-block:: json
 
         {
-            "intent": {"name": "greet", "confidence": 0.98343}
+            "intent": {"name": "greet", "confidence": 1.0}
         }
 
 :Description:
-    This classifier is mostly used as a placeholder. It is able to recognize `hello` and
-    `goodbye` intents by searching for these keywords in the passed messages.
+    This classifier works by searching a message for user-defined keywords.
+    The matching is case insensitive and search only for exact matches of the keyword-string.
+    This is intended to augment the other classifiers. In order to use it it must be
```

Comment:
I wouldn't say it's necesarily intended to augment other classifiers? Where did this come from?
And why is it dependent on other classifiers?

Reply:
I should have said something like "_only_ intended to augment others", as in this should not be the only classifier being used.
As far as I know there is nothing in the NLU that compares different classifiers, (is there?). Usually the pipeline has exactly one classifier. In that case the order is important because intent is set by the classifier changing a property of the message object. The order is which multiple classifiers do this is the order in which they are specified in the pipeline, so the keyword classifier must be last or it will have its intent overwritten.